lib/math.ts

```diff
@@ -55,3 +58,50 @@ export function FMAPowerOf10(xParam: number, p: number, zParam: number) {
   const resStr = xStr + ReflectApply(StringPrototypePadStart, zStr, [p, '0']);
   return sign * NumberParseInt(resStr, 10);
 }
+
+type UnsignedRoundingMode = 'half-even' | 'half-infinity' | 'half-zero' | 'infinity' | 'zero';
+
+export function GetUnsignedRoundingMode(
+  mode: Temporal.RoundingMode,
+  sign: 'positive' | 'negative'
+): UnsignedRoundingMode {
+  const index = +(sign === 'negative'); // 0 = positive, 1 = negative
+  switch (mode) {
+    case 'ceil':
+      return (['infinity', 'zero'] as const)[index];
```

Comment:
I don't love this complexity here. Seems like a conditional would be clearer, e.g. `return isNegative ? 'infinity' : 'zero'`.  If we're keeping it this way to align with upstream, then OK, but if not then simpler would be nicer!

Reply:
Makes sense. This is how upstream does it but I can't remember why I did it that way; probably to make it work more like looking it up in a table, which is how it's expressed in the spec. We already have to deviate from upstream to add `as const` to make it compile, so may as well use the conditional instead.

Reply:
Cool. I guess the other option would be to make it completely table-driven, like an object where the keys are the rounding mode and the values are either a single string or a tuple. Then we could forego the switch statement completely.